BookAPI/Program.cs

```diff
@@ -2,11 +2,27 @@
 
 namespace BookAPI
 {
-    class Program
+    public class Program
     {
         static void Main(string[] args)
         {
             Console.WriteLine("Hello World!");
         }
+
+        public string ValidateNewBookModel(BookModel bookModel)
+        {
```

Comment:
why is it returning a string?
If your feature is just validation, take a look at FluentValidation
https://github.com/JeremySkinner/FluentValidation